src/local/butler/reproduce_tool/constants.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Constants used by the reproduce tool."""
+
+import os
+
```

Comment:
Feel like more things can be moved here.

Reply:
I deleted this file for the time being. It might make sense to refactor some things and use this later, but I originally intended to use this for a few things in this change before realizing it was unnecessary.